lua/telescope/pickers.lua

```diff
@@ -490,6 +529,8 @@ function Picker:find()
 
   if self.initial_mode == "insert" then
     vim.cmd [[startinsert!]]
+    -- make starting in insert mode more robust
+    vim.cmd [[normal! A]]
```

Comment:
Is this a workaround for opening a picker in an action if telescope wasn't closed with `actions._close(prompt_bufnr, true)`

Reply:
Sort of, yes 😅 as I didn't recall `_close` vs `close` in `builtin.internal.pickers`.

Not sure whether to keep this (since it shouldn't hurt?) or only change to `_close` in pickers action.